meta-client/src/test/java/org/apache/helix/metaclient/recipes/lock/LockInfoTest.java

```diff
@@ -0,0 +1,50 @@
+package org.apache.helix.metaclient.recipes.lock;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+
+import java.time.Duration;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class LockInfoTest {
```

Comment:
Let's also test convert a ZNRecord to LockInfo

Reply:
Not sure if i understand this. If lockinfo is a wrapper around znrecord, how would one convert a znrecord to lockinfo?

Reply:
I meant this constructor `  public LockInfo(ZNRecord znRecord) `